packages/cli/package.json

```diff
@@ -34,10 +34,11 @@
     "prepublish": "",
     "prepack": "yarn run build && oclif readme",
     "homebrew": "node ./homebrew/scripts/prepare.mjs",
-    "test": "TZ=UTC NODE_OPTIONS='--experimental-vm-modules' yarn jest --runInBand --forceExit",
-    "test-anvil": "RUN_GANACHE_TESTS=false RUN_ANVIL_TESTS=true TZ=UTC NODE_OPTIONS='--experimental-vm-modules' yarn jest --forceExit",
+    "test": "TZ=UTC NODE_OPTIONS='--experimental-vm-modules' yarn jest --maxWorkers 50% --forceExit",
```

Comment:
care to explain the change?

Reply:
--runInBand was only needed because of ganache. test-anvil is likewise not needed since its now the same as test

im expirementing with the maxWorkers part. actually i have enough changes locally for an entirely new pr with just changes in our test commands across all the packages. tldr i was able to run all the tests locally in just 2:30 will open a new pr for those that changes this line again probably.

changed max workers so running tests doesnt use ALL our cores and we can still do stuff while they are running.